pkg/adapter/apiserver/delegate.go

```diff
@@ -44,6 +45,12 @@ func (a *resourceDelegate) Add(obj interface{}) error {
 
 	if result := a.ce.Send(context.Background(), event); !cloudevents.IsACK(result) {
 		a.logger.Errorw("failed to send event", zap.Error(result))
+	} else {
+		eventExt, err := json.Marshal(event.Extensions())
+		if err != nil {
+			a.logger.Error(err)
+		}
+		a.logger.Infof("event sent for resource %s", string(eventExt))
```

Comment:
I remember acknowledgement logs like this were in debug level in `eventing-kafka`, but other than that I am not sure whats the standard followed here. I personally think that an acknowledgement message for successful delivery of an event is primarily informative and not noise, but I could be wrong - let me know, I will change it. 
Yeah, as a user I'd only want to see the k8s event (the source here) for which the cloudevent has been sent and not the complete cloudevent itself, because that would be visible in the sink anyway.

Reply:
+1 for Debug or Trace, this could be a lot of log lines for a chatty cluster.